src/cluster/src/cli/start/k8.rs

```diff
@@ -34,6 +34,7 @@ pub async fn process_k8(
         .chart_values(opt.k8_config.chart_values)
         .render_checks(true)
         .upgrade(upgrade)
+        .use_cluster_ip(true)
```

Comment:
If your branch is based on mine then you should double-check if this is needed. I added this when I was still figuring out the different IP types.